testdrive/liquidity_by_token_pair/preprod/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  scrolls:
+    # change this back to official repo instead of local image/ fork repo
+    image: ghcr.io/txpipe/scrolls:latest
```

Comment:
Shouldn't this be the local image with your changes? :D

Reply:
Yes, I intentionally did not commit it because I thought it shouldn't differ from other test cases if we were to merge the fork back to txPipe. But I am happy to commit the local changes.

Reply:
At this moment we do not have docker image published yet (with Will's reducer), so I think it is not so important to have it updated. 

If we one day decide to contribute back to txpipe, we will have this testdrive / example of usage in place. We will just merge to their repo and it should generate the proper docker image from txpipe registry.

But..
As I see that Will mentioned this in the reducer's README.md file (`How to run`), maybe we should update the README, and put information that engineers need to build docker image on their local env and replace this `image` in `docker-compose.yml` before using it?

Reply:
I've changed it to the local image and added respective details on how to build it and run it via docker build + docker-compose up.